DEVELOPMENT.md

```diff
@@ -126,3 +126,39 @@ On Fedora, this can be fixed by adding a `.conf` file to `/etc/sysctl.d/ increas
 fs.inotify.max_user_watches=2097152
 fs.inotify.max_user_instances=256
 ```
+
+### Error: can only create exec sessions on running containers: container state improper
+
+This may happen when the containers have stopped. This can be fixed by deleting the already created clusters and starting the script again.
```

Comment:
> the containers have stopped

=> the kind container has stopped.
Note that kind containers are ephemeral and do not survive a reboot of the host.